sql/moz-fx-data-marketing-prod/google_search_console_derived/search_impressions_by_page_v2/metadata.yaml

```diff
@@ -24,19 +24,19 @@ scheduling:
   depends_on_table_partitions_existing:
   - task_id: wait_for_google_search_console_addons_url_impressions
     table_id: moz-fx-data-marketing-prod.searchconsole_addons.searchdata_url_impression
-    partition_id: '{{ macros.ds_add(ds, -1) | ds_nodash }}'
+    partition_id: '{{ data_interval_start.subtract(days=1) | ds_nodash }}'
```

Comment:
`macros.ds_add()` returns a string (not a datetime), and `ds_nodash` expects a datetime input.